src/Listeners/UptimeCheckFailedListener.php

```diff
@@ -5,15 +5,26 @@
 use Illuminate\Support\Facades\Notification;
 use Taecontrol\MoonGuard\Repositories\UserRepository;
 use Taecontrol\MoonGuard\Events\UptimeCheckFailedEvent;
+use Taecontrol\MoonGuard\Notifications\SlackNotifiable;
 use Taecontrol\MoonGuard\Notifications\UptimeCheckFailedNotification;
 
 class UptimeCheckFailedListener
 {
     public function handle(UptimeCheckFailedEvent $event): void
     {
-        Notification::send(
-            UserRepository::all(),
-            new UptimeCheckFailedNotification($event->uptimeCheck, $event->downtimePeriod)
-        );
+        $channels = config('moonguard.notifications.channels');
+
+        foreach ($channels as $channel) {
+            $users = ($channel === 'slack') ? new SlackNotifiable() : UserRepository::all();
```

Comment:
Rename this variable into:

```suggestion
            $notifiables = ($channel === 'slack') ? new SlackNotifiable() : UserRepository::all();
```